oci/archive/oci_transport.go

```diff
@@ -174,8 +177,8 @@ func createOCIRef(image string) (tempDirOCIRef, error) {
 }
 
 // creates the temporary directory and copies the tarred content to it
-func createUntarTempDir(ref ociArchiveReference) (tempDirOCIRef, error) {
-	tempDirRef, err := createOCIRef(ref.image)
+func createUntarTempDir(ref ociArchiveReference, sys *types.SystemContext) (tempDirOCIRef, error) {
```

Comment:
By convention, the parameters go `context.Context, *types.SystemContext`, then others.